data/jakarta/test-security-services-proxy-certs.sh

```diff
@@ -108,14 +108,15 @@ snap_wait_port_status 50025 open
 # We are running the test script with 'sudo' and although the edgeca snap has the home interface, 
 # which allows access to the home directory, when running as sudo, the user is root, 
 # so it has a different home directory and doesn't have write access to your home directory. 
-# The simplest fix: use `su - "$USER" -c "command"`
-su - "$USER" -c "edgeca gencsr --cn localhost --csr csrfile --key csrkeyfile"
-su - "$USER" -c "edgeca gencert -o localhost.cert -i csrfile -k localhost.key"
-snap set edgexfoundry env.security-proxy.tls-certificate="$(cat localhost.cert)"
-snap set edgexfoundry env.security-proxy.tls-private-key="$(cat localhost.key)"
-
-# verify CA-signed TLS certificate
-code=$(curl --insecure --silent --include \
+# It's therefore easiest to use the $SNAP_DATA directory of the EdgeCA snap:
+EDGECA_DIR="/var/snap/edgeca/current/"
```

Comment:
EDGECA_DIR="/var/snap/edgeca/current ~~/~~". 
This "/" could be removed, otherwise, the log will be "Wrote Certificate to /var/snap/edgeca/current//localhost.cert"

Reply:
done.